setup.cfg

```diff
@@ -79,7 +79,7 @@ all =
     ipython>=4.2
     pytest
 docs =
-    sphinx<4
+    sphinx<4,>=4.0.2
```

Comment:
Is the syntax wrong, @saimn ?

Reply:
Yeah, this is the empty set! If anything, it shou d say Sphinx!=4.0.0,!=4.0.1, right? (On the smartphone, not 100 % sure of the syntax)

Reply:
Or... I thought it is `or`. Don't know why I think that. Thanks!

Reply:
Maybe I'll just remove the pin altogether. Not worth the complicated syntax.